docs/source/linalg.rst

```diff
@@ -80,6 +80,7 @@ Tensor Operations
 
     tensorinv
     tensorsolve
+    tensordot
```

Comment:
Nit. Probably put it before `tensorinv` and `tensorsolve` as it's the main operation

Reply:
done af85b06e0b